Input/IME/WinForms/WinFormsIMEHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ImeSharp;
+
+using Microsoft.Xna.Framework;
+
+namespace Rampastring.XNAUI.Input.IME.WinForms
+{
+    /// <summary>
+    /// Integrate IME to XNA framework.
+    /// </summary>
+    internal class WinFormsIMEHandler : IMEHandler
+    {
+        public WinFormsIMEHandler(Game game)
+        {
+            InputMethod.Initialize(game.Window.Handle);
+            InputMethod.TextInputCallback = OnTextInput;
+            InputMethod.TextCompositionCallback = (compositionText, cursorPosition, _, _, _, _) =>
+            {
+                Composition = compositionText.ToString();
+                CompositionCursorPos = cursorPosition;
+            };
+        }
+
+        public override bool Enabled
+        {
+            get => InputMethod.Enabled;
+            protected set => InputMethod.Enabled = value;
+        }
+
+        public override void StartTextComposition()
+        {
+            Enabled = true;
+        }
+
+        public override void StopTextComposition()
+        {
+            Enabled = false;
+        }
+
+        public override void SetTextInputRect(in Rectangle rect)
+        {
+            InputMethod.SetTextInputRect(rect.X, rect.Y, rect.Width, rect.Height);
+        }
```

Comment:
Should use expression bodies here.

Reply:
> Should use expression bodies here.

We should provide a .editorconfig file to tell contributors what kind of code specification they should follow.

Without a .editorconfig, my VS warns me about expression bodies, file scope namespaces, and these syntax hints.

![image](https://github.com/Rampastring/Rampastring.XNAUI/assets/42184238/eb700038-b37e-40d8-a255-4b55c13ca392)